### ISSUE TITLE
Missing `main` field in package.json

### ISSUE THREAD

**Issue by fritx** [Gozala/undefined.js#1]
Hey guy i find a `main` field seems missing, and wonder if this package could work in Node...

**Comment by skozin** [comment]
Yup, it would be great if you added the `main` field to `package.json`, pointing to `undefined.js`, or rename `undefined.js` to `index.js`. Currently, to require this module, on needs to use `undefined/undefined`, which seems too explicit :)

BTW, despite the "troll" tag in `package.json`, this module actually has real uses =) For example, when building for produciton, I'm mapping development-time dependencies of the app to this module using webpack's `alias` config option, which prevents them from being compiled into the script bundle.

**Comment by Gozala** [comment]
Pull requests are accepted :)

**Comment by antoniobrandao** [comment]
I install globally, works great.

**Comment by devramsean0** [comment]
#6
